### ISSUE TITLE
Php bindings don't build for php 7

### ISSUE THREAD

**Issue by andreasbaumann** [patrickfrey/strusBindings#14]
Php7 support in swig missing upstream, see https://github.com/swig/swig/issues/571, so
building Php bindings on ArchLinux with php5 is the only option.

**Comment by andreasbaumann** [comment]
Also seen with php 5 on ArchLinux:

```
 # pragma message("NOTE: Use of this header (ice_not.hpp) is deprecated")
                                                                        ^
In file included from /usr/include/boost/type_traits/ice.hpp:18:0,
                 from /usr/include/boost/python/detail/def_helper.hpp:9,
                 from /usr/include/boost/python/class.hpp:29,
                 from /usr/include/boost/python.hpp:18,
                 from /home/abaumann/projects/strus/strusBindings/lang/python/strusPythonModule.cpp:33:
/usr/include/boost/type_traits/detail/ice_eq.hpp:17:71: note: #pragma message: NOTE: Use of this header (
 # pragma message("NOTE: Use of this header (ice_eq.hpp) is deprecated")
```

This is also a problem of the Swig-Templates for Php, I think.

**Comment by andreasbaumann** [comment]
They do buld against php56 from AUR.

**Comment by andreasbaumann** [comment]
Ubuntu 16.04 and 16.10 also have php7, no php56 bindings currently.

**Comment by andreasbaumann** [comment]
There is support for Php7 available for Swig 3.0.11, so we need modern Swig on almost all of the OBS platforms..

**Comment by patrickfrey** [comment]
We switched to PHP 7, PHP 5.6 is not supported anymore.
